.github/workflows/lint.yaml

```diff
@@ -0,0 +1,14 @@
+name: YAPF Formatting Check
+
+on: [push]
+
+jobs:
+  formatting-check:
+    name: Formatting Check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: run YAPF to test if python code is correctly formatted
+      uses: AlexanderMelde/yapf-action@master
```

Comment:
Might be [safer](https://julienrenaux.fr/2019/12/20/github-actions-security-risk/) to use the commit hash versioning rather than pointing to a branch, no?

Also, why use a 3rd-party action rather than just pip-installing YAPF within the workflow and running YAPF on the repo?

Reply:
I largely agree if this were anything more than a YAPF test, but given that it seems like a pretty low risk thing to do. 

_Eventually_ Dependabot is supposed to support action updates as well so we can reevaluate then.